code/solr/services/SolrService.php

```diff
@@ -10,7 +10,7 @@
  */
 class SolrService extends SolrService_Core
 {
-    private static $core_class = 'SolrService_Core';
+    private static $core_class = 'SilverStripe\FullTextSearch\Solr\Services\SolrService_Core';
```

Comment:
use `SolrService_Core::class` instead. It may not work since it's a static variable